runtime/vm/ContinuationHelpers.cpp

```diff
@@ -286,4 +286,60 @@ walkContinuationStackFrames(J9VMThread *currentThread, J9VMContinuation *continu
 
 	return rc;
 }
+
+UDATA
+walkAllStackFrames(J9VMThread *currentThread, J9StackWalkState *walkState)
+{
+	Assert_VM_mustHaveExclusiveVMAccess(currentThread);
```

Comment:
you need to assert exclusive or safepoint

`Assert_VM_true((J9_XACCESS_EXCLUSIVE == vm->exclusiveAccessState) || (J9_XACCESS_EXCLUSIVE == vm->safePointState));`

Reply:
fixed